src/gameGlobalInfo.h

```diff
@@ -76,6 +76,8 @@ class GameGlobalInfo : public MultiplayerObject, public Updatable
 
     //List of script functions that can be called from the GM interface (Server only!)
     std::list<GMScriptCallback> gm_callback_functions;
+    //List of names of gm_callback_functions scripts (replicated to clients)
```

Comment:
I understand why this list is here, but now we have duplicate data, name is in both std::vectors. Would be cleaner to remove the name from the GMScriptCallback (which might not need to exist then and can be just a ScriptSimpleCallback object?)

Reply:
http://www.cplusplus.com/reference/algorithm/remove_if/

Reply:
nevermind, I just RTFM-ed.